xsrc/test/scripts/refactor/fix-imports.scala

```diff
@@ -0,0 +1,184 @@
+import java.io.BufferedWriter
+import java.io.FileWriter
+import java.io.File
+import scala.collection.mutable.ListBuffer
+import scala.collection.mutable.Map
+import scala.io.Source
+
+object FixImports extends App {
+  val inline = true
+  val importMap = Map(
+    "daffodil-cli" -> "cli",
+    "daffodil-core" -> "core",
+    "daffodil-io" -> "io",
+    "daffodil-lib" -> "lib",
+    "daffodil-macro-lib" -> "lib",
+    "daffodil-runtime1" -> "runtime1",
+    "daffodil-runtime1-layers" -> "layers.runtime1",
+    "daffodil-runtime1-unparser" -> "unparsers.runtime1"
```

Comment:
Maybe runtime1 wants to be `org.apache.daffodil.runtime1.core` or `org.apache.daffodil.runtime1.parsers`? That way all the runtime1 stuff are in subpackages of the same parent package.

Reply:
Core is the short name for the primary "schema compiler" module currently. I think using core for the compiler aspects of other modules is fine.

Maybe everything "core" should be renamed "comp" instead though.

Reply:
Both `daffodil-runtime1` and `daffodil-runtime1-layers` contain an `org.apache.daffodil.layers` package. Maybe `daffodil-runtime1` stays as-is and `daffodil-runtime1-layers` becomes `org.apache.daffodil.layers.impl`?

Reply:
The implementation of layers is in daffodil-runtime1
The predefined layers that come built-in to Daffodil are in daffodil-runtime1-layers. 

So I would think daffodil-runtime1's layers should be the one that becomes layers.impl.

Reply:
Selecting a good name is always a challenging task. I'm neutral on the matter. Out of these three (or with another name not yet listed), let me know which one we should use and I'll update the scripts .